### PR TITLE
docs: add McDonald's section

### PR DESCRIPTION
**Pull request by himself65**
Refs: https://github.com/troll-busters/himself65-way/pull/2#issuecomment-1798521488

### CODE REVIEW

**Comment by suica** [comment]
Just out of curiosity, what will be the most delicious drink, in your opinion?

**Comment by himself65** [comment]
> Just out of curiosity, what will be the most delicious drink, in your opinion?

Sprite

Refs: https://twitter.com/himself_65/status/1696015716317630769

File: README.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 PR welcome
 
+## 麦当劳篇
+
+![mcdonald's](https://github.com/troll-busters/himself65-way/assets/14026360/f73a96d6-3518-4099-bfae-4e83332099e5)
```
Comment (ghost) [inline README.md]: Please make the image smaller

File: README.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+
+> 麦当劳可乐沙冰绝对是第二好喝的东西了
+
+- https://twitter.com/himself_65/status/1654669861069418496
```
Comment (ghost) [inline README.md]: Archive the page in https://web.archive.org/save and add the archive link

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**